pages/advocates/index.vue

```diff
@@ -1,22 +1,6 @@
 <template>
   <main class="community-page advocates-page">
-    <header>
-      <CommunityHeader
-        id="presentation"
-        main-title="Qiskit Advocates"
-      >
-        <p>A global program that provides support to the individuals who actively contribute to the Qiskit Community.</p>
-        <template #features>
-          <div id="advocates-benefits">
-            <CompactFeature
-              v-for="benefit in advocateBenefits"
-              :key="`benefit-${benefit.icon}`"
-              v-bind="benefit"
-            />
-          </div>
-        </template>
-      </CommunityHeader>
-    </header>
+    <TheAdvocatesHeader />
```

Comment:
Not a blocker, it is just in case you don't know it: We have [this issue](https://github.com/Qiskit/qiskit.org/issues/1022) to talk about how we should name the components. From my point of view, this should be `AdvocatesHeader`, but we should discuss it together :)